packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -131,7 +133,10 @@ export default function Sidebar(): ReactElement {
                 {key}
               </NavHeader>
               {items.map((item) => (
-                <NavItem key={item.title}>
+                <NavItem
+                  key={item.title}
+                  active={item.active || item.path === router?.asPath}
```

Comment:
Again we should support the extension as well here. You might want to think about getting this as a property from the outside. I mean the `router.asPath` part